src/content/12/en/part12a.md

```diff
@@ -29,19 +26,19 @@ Cloud services like AWS, Google Cloud, and Microsoft Azure all support container
 
 So containers are used in cloud environment and even during development. What are the benefits of using containers? Here are two common scenarios:
 
-<i>Scenario 1: You are developing a new application that needs to run on the same machine as a legacy application. Both require different versions of Node installed.</i>
+<i>Scenario 1: You are developing a new application that needs to run on the same machine as a legacy application. Both require installing different versions of Node.</i>
 
 You can probably use nvm, virtual machines, or dark magic to get them running at the same time. However, containers are an excellent solution as you can run both applications in their respective containers. They are isolated from each other and do not interfere.
 
 <i>Scenario 2: Your application runs on your machine. You need to move the application to a server.</i>
 
 It is not uncommon that the application just does not run on the server despite it works just fine on your machine. It may be due to some missing dependency or other differences in the environments. Here containers are an excellent solution since you can run the application in the same execution environment both on your machine and on the server. It is not perfect: different hardware can be an issue, but you can limit the differences between environments.
 
-Sometimes you may hear about the <i>"Works in my container"</i> issue. The phrase describes a situation in which the application works fine in a container running on your machine but breaks when the container is started on a server. The phrase is a play on the infamous <i>"Works on my machine"</i> issue, which containers are often promised to solve. The situation also is most likely a usage error.
+Sometimes you may hear about the <i>"Works in my container"</i> issue. The phrase describes a situation in which the application works fine in a container running on your machine but breaks when the container is started on a server. The phrase is a play on the infamous <i>"Works on my machine"</i> issue, which containers are often promised to solve. The situation also is most likely, an usage error.
```

Comment:
```suggestion
Sometimes you may hear about the <i>"Works in my container"</i> issue. The phrase describes a situation in which the application works fine in a container running on your machine but breaks when the container is started on a server. The phrase is a play on the infamous <i>"Works on my machine"</i> issue, which containers are often promised to solve. The situation also is most likely a usage error.
```

Reply:
It should be `The situation also is most likely a usage error.`.
Because it's not about the letter but the sound.

UK  /ˈjuː.sɪdʒ/
US  /ˈjuː.sɪdʒ/
> *source* [Cambridge dictionary](https://dictionary.cambridge.org/pronunciation/english/usage)


A usage is a meaning that a word has or a way in which it can be used.
It's very definitely a usage which has come over to Britain from America
> *source* [Collins dictionary](https://www.collinsdictionary.com/dictionary/english/usage)

Please refer:
- [a/an - Merriam-Webster](https://www.merriam-webster.com/grammar/is-it-a-or-an)